### PR TITLE
Improve Kinesis documentation

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/kinesis.rst [hunk 1]
```diff
@@ -1,159 +1,195 @@
...
 stored on Amazon S3 (preferred) or stored in a local directory on each Presto node.
 
-This connector is a read-only connector. It can only fetch data from kinesis streams, but cannot create streams or push
-data into existing streams.
+This connector is a read-only connector. It can only fetch data from Kinesis streams,
+but cannot create streams or push data into existing streams.
 
-To configure the Kinesis connector, create a catalog properties file, ``etc/catalog/CATALOG_NAME.properties`` with the following
-content and replace the properties as appropriate:
+To configure the Kinesis connector, create a catalog properties file ``etc/catalog/kinesis.properties``
```
Comment (martint) [inline presto-docs/src/main/sphinx/connector/kinesis.rst]: This should probably say “for example”. We don’t want to imply that the file *has* to be named exactly that.

File: presto-docs/src/main/sphinx/connector/kinesis.rst [hunk 2]
```diff
@@ -1,159 +1,195 @@
...
+
+The following configuration properties are available:
+
+=================================  =======================================================================
+Property Name                      Description
+=================================  =======================================================================
+``kinesis.access-key``             Access key to aws account or blank to use default provider chain
+``kinesis.secret-key``             Secret key to aws account or blank to use default provider chain
+``kinesis.aws-region``             AWS region to be used to read kinesis stream from
+``kinesis.default-schema``	       Default schema name for tables
```
Comment (martint) [inline presto-docs/src/main/sphinx/connector/kinesis.rst]: Extra leading space

File: presto-docs/src/main/sphinx/connector/kinesis.rst (hunk 1 above)
Comment (electrum) [reply presto-docs/src/main/sphinx/connector/kinesis.rst]: This is the language we use for other connectors. We explain that catalogs can be named anything in the main deployment documentation.

File: presto-docs/src/main/sphinx/connector/kinesis.rst (hunk 2 above)
Comment (electrum) [reply presto-docs/src/main/sphinx/connector/kinesis.rst]: It was actually using tabs. Not sure why IntelliJ didn't fix those.

**Pull request merged**